File: Python program to transfer Algorithms To MySQL/programs/Factorial/Factorial.cs
// C# code to implement factorial
using System;
class GFG {

// Factorial function
static int f(int n)
{
	// Stop condition
	if (n == 0 || n == 1)
	return 1;

	// Recursive condition
	else
	return n * f(n - 1);
}

// Driver code
static void Main()
{
	int n = 5;
	Console.WriteLine("factorial of " + n + " is: " + f(n));
}
}

// This code is contributed by divyeshrabadiya07.

File: Python program to transfer Algorithms To MySQL/programs/Factorial/Factorial.js
<script>
// JavaScript code to implement factorial

// Factorial function
function f(n)
{
// Stop condition
if(n == 0 || n == 1)
	return 1;
	
// Recursive condition
else
	return n*f(n-1);
}

// Initialize variable n.
let n = 5;
document.write("factorial of "+ n +" is: " + f(n));

// This code is contributed by probinsah.
</script>
